### PR TITLE
fix: add kustomize rollback to 1.56.0 rn

### PR DESCRIPTION
**Pull request by DanStough**

### CODE REVIEW

File: content/release-notes/1.56.0.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ kubernetes: "1.19, 1.20, and 1.21"
 {{</changes>}}
 
 {{<fixes>}}
+* Rolled back the version of Kustomize from 3.10.0 to 3.8.10 to fix errors related to duplicate YAML keys.
```
Comment (JustinN-Replicated) [inline content/release-notes/1.56.0.md:18]: * Reverted the version of Kustomize from 3.10.0 to 3.8.10 to fix errors related to duplicate YAML keys.

Comment (ajp-io) [reply content/release-notes/1.56.0.md:18]: I'm partial to rolled back in this case.

Comment (samantha-replicated) [inline content/release-notes/1.56.0.md:18]:
```suggestion
* To fix errors related to duplicate YAML keys, the version of Kustomize was reverted to 3.8.10 from 3.10.0.
```

Comment (samantha-replicated) [reply content/release-notes/1.56.0.md:18]: That's my suggestion, I also prefer Reverted.

**Review by JustinN-Replicated — APPROVED** [review]

**Pull request merged**